src/explorer/WebAppTreeItem.ts

```diff
@@ -126,3 +127,20 @@ export class WebAppTreeItem extends SiteTreeItem {
         return await fs.readFile(templatePath, 'utf8');
     }
 }
+
+export class InvalidWebAppTreeItem implements IAzureTreeItem {
```

Comment:
Please move this to a separate file "InvalidWebAppTreeItem.ts"

Reply:
I included it here, as that's the same way the `DeploymentSlotsNATreeItem` is defined in the `DeploymentSlotsTreeItem.js`... there's a precedent for doing it like so... should I still move it to a different file?

Reply:
Yeah @nturinski did that, not me 😊 I generally prefer one exported class per file, since you never know how much a class will grow as time goes on

That being said - this is kind of a stylistic preference and I wouldn't block a PR just for this

Reply:
Went ahead and moved it to it's own file (I prefer it that way as well, lol).